pages/form-page.php

```diff
@@ -23,106 +23,143 @@
-		<?php if ( $action == 'edit' ) : ?>
-		<input type="hidden" name="action" value="consentwow_form_post" />
-		<input name="consentwow_form[id]" type="hidden" value="<?php echo $id; ?>" />
-		<?php endif; ?>
-		<table class="form-table" role="presentation">
-			<tbody>
-				<tr class="consentwow-form-name-input">
-					<th scope="row">
-						<label for="consentwow_form_name">Form Name</label>
-					</th>
-					<td>
-						<input
-							required
-							type="text"
-							id="consentwow_form_name"
-							name="consentwow_form[form_name]"
-							class="regular-text"
-							value="<?php echo isset( $form['form_name'] ) ? $form['form_name'] : ''; ?>"
-						/>
-					</td>
-				</tr>
-				<tr class="consentwow-form-id-input">
-					<th scope="row">
-						<label for="consentwow_form_id">Form ID</label>
-					</th>
-					<td>
-						<input
-							required
-							type="text"
-							id="consentwow_form_id"
-							name="consentwow_form[form_id]"
-							class="regular-text"
-							value="<?php echo isset( $form['form_id'] ) ? $form['form_id'] : ''; ?>"
-						/>
-					</td>
-				</tr>
-				<tr class="consentwow-email-input">
-					<th scope="row">
-						<label for="consentwow_email">Email</label>
-					</th>
-					<td>
-						<input
-							required
-							type="text"
-							id="consentwow_email"
-							name="consentwow_form[email]"
-							class="regular-text"
-							value="<?php echo isset( $form['email'] ) ? $form['email'] : ''; ?>"
-						/>
-					</td>
-				</tr>
-				<tr class="consentwow-first-name-input">
-					<th scope="row">
-						<label for="consentwow_first_name">First Name</label>
-					</th>
-					<td>
-						<input
-							type="text"
-							id="consentwow_first_name"
-							name="consentwow_form[first_name]"
-							class="regular-text"
-							value="<?php echo isset( $form['first_name'] ) ? $form['first_name'] : ''; ?>"
-						/>
-					</td>
-				</tr>
-				<tr class="consentwow-last-name-input">
-					<th scope="row">
-						<label for="consentwow_last_name">Last Name</label>
-					</th>
-					<td>
-						<input
-							type="text"
-							id="consentwow_last_name"
-							name="consentwow_form[last_name]"
-							class="regular-text"
-							value="<?php echo isset( $form['last_name'] ) ? $form['last_name'] : ''; ?>"
-						/>
-					</td>
-				</tr>
-				<tr class="consentwow-phone-number-input">
-					<th scope="row">
-						<label for="consentwow_phone_number">Telephone No.</label>
-					</th>
-					<td>
-						<input
-							type="text"
-							id="consentwow_phone_number"
-							name="consentwow_form[phone_number]"
-							class="regular-text"
-							value="<?php echo isset( $form['phone_number'] ) ? $form['phone_number'] : ''; ?>"
-						/>
-					</td>
-				</tr>
-			</tbody>
-		</table>
-		<p class="submit">
-			<input type="submit" name="submit" id="submit" class="button button-primary" value="Save Changes">
-		</p>
-	</form>
+    <h1 class="wp-heading-inline"><?php echo $title ?></h1>
```

Comment:
```suggestion
	<h1 class="wp-heading-inline"><?php echo $title ?></h1>
```
This one also have to [use tabs and not spaces](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/html/#indentation)